meetings/osf_oauth2_adapter/migrations/0001_add_site_and_social_app.py

```diff
@@ -7,6 +7,8 @@
 from allauth.socialaccount.models import SocialApp
 from django.contrib.sites.models import Site
 
+from osf_oauth2_adapter.apps import OsfOauth2AdapterConfig
+
 CLIENT_ID = "c8afa24def0c47b6a261f6e431891d44"
 CLIENT_SECRET = "9px3QS1DNcUNcnQEnW1RgnCutVoZcTDnFafnCIwO"
```

Comment:
These cannot be here, CLIENT_ID and SECRET should be set in settings, most likely in `local.py` as setup with #31. Is this a hand-edited migration file? If it is we might need to denote it, at least for this initial release, after that we will just add new migrations files rather than editing old ones.